src/hugr/serialize.rs

```diff
@@ -35,7 +36,7 @@ struct SerHugrV0 {
     /// For each node: (parent, num_inputs, num_outputs, node_operation)
     nodes: Vec<(NodeIndex, usize, usize, OpType)>,
     /// for each edge: (src, src_offset, tgt, tgt_offset)
```

Comment:
```suggestion
    /// for each edge: (src, src_subport_offset, tgt, tgt_subport_offset)
```
?

Reply:
We don't encode the specific subports, just the port in the operation node.
A serialization-deserialization roundtrip could potentially reorder the subports without changing the interpretation of the graph (In practice that won't happen because we encode things sequentially).